lib/methods/import.js

```diff
@@ -82,7 +82,8 @@ module.exports = self => {
         event: {
           name: 'import-duplicates',
           data: results
-        }
+        },
+        classes: [ 'apos-notification--hidden' ]
```

Comment:
I think you could deleted the translation key that were used here, and simply pass an empty string as second parameter?

Reply:
translation key is also used in the modal